app/models/contribution.rb

```diff
@@ -11,4 +11,14 @@ def self.get_nearby(lat, long, dist)
                       .where('contributor_location_1_long >= ?', long_ranges[:min_long])
     contributions
   end
+
+  def lat_lng
+    lat_lng = []
+    lat_lng.push(contributor_location_1_lat).push(contributor_location_1_long)
+  end
+
```

Comment:
Let's test this by using FactoryBot to mock up a contribution and then make sure that it returns the expected value.

Reply:
Okay...